### DISCUSSION TITLE
how to show shortcuts

### DISCUSSION THREAD

**Discussion by cstkingkey** [fltk-rs/fltk-rs#1164] [Q&A]
nativefileschooser browsedir type doesn't show shortcuts in desktop under Windows
It seems that it uses folderbrowerdialog. Can it use openfiledialog?

**Comment by MoAlyousef** [comment]
Not thru FLTK's public api. Is BrowseFile not appropriate for your use case?

**Reply by cstkingkey** [reply]
I need to use dirs from smb servers. Currently I cannot do that.
